### PR TITLE
en_tanron6 OK

### PR DESCRIPTION
**Pull request by fullgrowngaming**

### CODE REVIEW

**Review by Rozelette — APPROVED** [review]
LGTM

File: src/overlays/actors/ovl_En_Tanron6/z_en_tanron6.c [hunk 1]
```diff
@@ -21,16 +23,39 @@ const ActorInit En_Tanron6_InitVars = {
...
+    this->actor.unkA0.damageChart = &D_80BE6170;
+    this->actor.unk1F = 6;
+    EnTanron6_SetupDoNothing(this);
+}
+
+void EnTanron6_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+}
 
-GLOBAL_ASM("asm/non_matchings/ovl_En_Tanron6_0x80BE6040/EnTanron6_Destroy.asm")
+void EnTanron6_SetupDoNothing(EnTanron6* this) {
```
Comment (Kenix3) [inline src/overlays/actors/ovl_En_Tanron6/z_en_tanron6.c:45]: Not a fan of this function name. EnTanron6_SetupActionFunc is better.

**Review by Kenix3 — CHANGES_REQUESTED** [review]

**Comment by fullgrowngaming** [comment]
I had it that originally, but the OoT people told me that was only used in cases where an actionfunc was actually passed into the "setupaction" function, and otherwise, it's named along these lines

**Comment by Kenix3** [comment]
> 
> 
> I had it that originally, but the OoT people told me that was only used in cases where an actionfunc was actually passed into the "setupaction" function, and otherwise, it's named along these lines

It doesn't DoNothing, so the name is very bad. If the naming comes from the OOT people, then we need to have a conversation with them.

If there's no better name, then it should be named back to "func_80BE60BC" for now.

File: src/overlays/actors/ovl_En_Tanron6/z_en_tanron6.c (hunk 1 above)
Comment (Kenix3) [reply src/overlays/actors/ovl_En_Tanron6/z_en_tanron6.c:45]: I made a mistake here. This doesn't accept a function pointer. It just sets a statically defined function to the actionFunc.

File: src/overlays/actors/ovl_En_Tanron6/z_en_tanron6.c [hunk 2]
```diff
@@ -9,7 +9,9 @@ void EnTanron6_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnTanron6_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnTanron6_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void EnTanron6_SetupDoNothing(EnTanron6* this);
+void func_80BE60D0(EnTanron6* this, GlobalContext* globalCtx);
```
Comment (Kenix3) [inline src/overlays/actors/ovl_En_Tanron6/z_en_tanron6.c:13]: This should be named EnTanron6_DoNothing

**Review by Kenix3 — CHANGES_REQUESTED** [review]
I made a mistake in the initial review. I thought the actionFunc was being set to a function pointer argument rather than a statically defined function.

I do still think it's unclear that it sets up the "DoNothing" action, so I would suggest to name the "DoNothing" function at the same time.

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**